civihr_default_theme/scss/components/_header.scss

```diff
@@ -124,17 +128,19 @@
   display: flex;
 
   a {
-    border-bottom: 1px solid $crm-grizzly;
-    color: $gray-darker;
-    display: block;
-    margin: 0 25px;
-    padding: 0 20px;
+    border: 0;
+    color: inherit;
+    margin: 0;
+    padding: 0 10px;
```

Comment:
Why the padding has changed from 20px to 10px? Is it an instruction from a designer or does it just look better?

Also, please have a look if we already have a variable for `0 10px` and what is it for. If there is no such a var, then please consider creating it.

Reply:
> Why the padding has changed from 20px to 10px? Is it an instruction from a designer or does it just look better?
Previously the styles for mobile was declared here, and for desktop we had `@media(min-width: $screen-md-max) {` block below. But as per your previous comment(which I agree with) we should define the desktop rules first and have media queries for other devices. So I have moved around the styles, but the end result is still the same.

Reply:
Also, the value is not used anywhere in this partial. So not creating variables.